products.js

```diff
@@ -238,7 +238,7 @@ function tunedPriceFnodes(nodeToCssMap, coeffDollarSign = 4.4, coeffNearDollarSi
     }
 
     function semanticTags(fnode){
-      if (fnode.element.getElementsByTagName('SUP').length > 0 || fnode.element.getElementsByTagName('STRONG') > 0){
+      if (fnode.element.getElementsByTagName('SUP').length > 0){
```

Comment:
In the case of a substantial change like this, I'd want to see a rationale in the commit message. This helps the reviewer and anybody who runs `git blame` someday.